assets/src/stories-editor/components/editor-carousel/reordererIndicator.js

```diff
@@ -0,0 +1,185 @@
+/**
```

Comment:
Why was this renamed from `indicator.js` to `reordererIndicator.js`?

We do not really use such naming for other components and their files. If the component is named `ReordererIndicator`, then `reorderer-indiccator.js` would be more fitting.

However, I don't see why it was renamed. Sure, the component now has a new job, but this name change seems misleading and unneeded IMO.

Reply:
@swissspidy Agree on this, I will rename it back to `indicator.js`